Assets/GothicVR/Scripts/Creator/WorldCreator.cs

```diff
@@ -24,8 +24,12 @@ public class WorldCreator : SingletonBehaviour<WorldCreator>
         private GameObject teleportGo;
         private GameObject nonTeleportGo;
 
+        public bool IsWorldLoaded { get; private set; }
```

Comment:
You don't need this. I added UnityEvents for WorldLoading inside this branch: https://github.com/GothicVRProject/GothicVR/pull/201

(As we need to set CullingGroup data once the scene is there). Feel free to check this branch first and reintegrate into your branch second.

Reply:
Bonus thing: You can also use the Events for "Unloading" worlds and "stop the MusicUpdate stuff etc.

Reply:
Will update to use the UnityEvent once #201 is merged into main